docs/guides/documentdb/failure-and-disaster-recovery/failover.md

```
@@ -0,0 +1,166 @@
+title: DocumentDB Automatic Failover
+menu:
+  docs_{{ .version }}:
+    identifier: dc-failure-disaster-recovery-failover
+    name: Automatic Failover
+    parent: dc-failure-disaster-recovery
+    weight: 10
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)